clusters.yml

```diff
@@ -20,6 +20,18 @@
       when: my_cluster_id is defined
 
     - name: List deleted cluster
-      debug:
+      ansible.builtin.debug:
         var: delete_cluster
       when: delete_cluster is defined
+
+    - name: Retrieve a cluster
+      vars:
+      clusters:
+        action: get
+        with_hosts: true
+        cluster_id: 1ac5ec14-dc29-4c34-8013-29f8de0ccc00
```

Comment:
Please use a var and add a when condition to only call this task when that ID is defined, if you could do the same for the debug and the delete while at it, I'll appreciate it much :smile_cat: